packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.service.js

```diff
@@ -49,7 +54,7 @@ export default class {
       .query().$promise;
   }
 
-  getServices(count, offset, search, type, renewDateType, status, order, nicBilling) {
+  getServices(count, offset, search, type, renewDateType, status, state, order, nicBilling) {
```

Comment:
Hmm no eslint error for line-length (I know the line length was already above the limit) ?

Reply:
No I didn't get an error. I will check my config though